config/settings.yml

```diff
@@ -250,6 +250,11 @@ vet360:
     enabled: true
     timeout: 30
     mock: false
+  veteran_status:
```

Comment:
Let's make sure to put these values in the manifests repo as well to keep things uniform. It will typically pull from this settings.yml if not redefined, but for easier understanding, we may want redefine them in the manfiests

Reply:
Manifest PR here - https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2418

Reply:
Sounds good!